documentationWebsite/docs/client-configuration.md

```diff
@@ -15,7 +15,7 @@ let client = {
     ~cache=Cache.InMemoryCache.make(),
     // I would turn this off in production
     ~connectToDevTools=true,
-    ~uri="https://graphql.org/swapi-graphql"
+    ~uri="https://graphql.org/swapi-graphql",
```

Comment:
In practise with `v1.1.0` you need to say:
```suggestion
    ~uri="https://graphql.org/swapi-graphql"->Obj.magic,
```

Reply:
Ah, good catch. It should indeed be a `UriFunction.t` like `~uri=_ =>"https://graphql.org/swapi-graphql",` In general, when there are multiple options and one of them could express all the others, I use that rather than require some variant like `String(string) | UriFunction(UriFunction.t)`

Reply:
```suggestion
    ~uri=_ => "https://graphql.org/swapi-graphql",
```